Documentation/ApiOverview/Logging/Writers/Index.rst

```diff
@@ -172,15 +172,17 @@ the :ref:`PhpErrorLog <logging-writers-php>` writer).
 Usage in Custom Class
 ---------------------
 
-All log writers can be used in your own classes. You can initialize the loggers in the __construct method of your class :code:`\MyDomain\MyExtension\MyFolder\MyClass`.
+All log writers can be used in your own classes. If the service is configured to use autowiring 
+you can initialize the loggers in the __construct method of your class :code:`\MyDomain\MyExtension\MyFolder\MyClass`) . 
+If no autowiring is used the service class however must implement the interface :code:`\Psr\Log\LoggerAwareInterface` and :code:`\Psr\Log\LoggerAwareTrait` and use no constructor.
```

Comment:
```suggestion
If autowiring is disabled, the service class however must implement the interface :php:`\Psr\Log\LoggerAwareInterface` and use :php:`\Psr\Log\LoggerAwareTrait`.
```

"and use no constructor" is misleading, as you can use a constructor to inject additional dependencies.

I also suggest to add an example for this case.